src/test/run-make-fulldeps/coverage-reports/expected_show_coverage.continue.txt

```diff
@@ -0,0 +1,75 @@
+    1|       |#![allow(unused_assignments, unused_variables)]
+    2|       |
+    3|      1|fn main() {
+    4|      1|    let is_true = std::env::args().len() == 1;
+    5|      1|
+    6|      1|    let mut x = 0;
+    7|     11|    for _ in 0..10 {
+                      ^10
+    8|     10|        match is_true {
+    9|       |            true => {
+   10|     10|                continue;
+   11|       |            }
+   12|      0|            _ => {
+   13|      0|                x = 1;
+   14|      0|            }
```

Comment:
small detail, but can you think of why the beginning/end of the block have counters for the `_` / `x = 1` case, but not the continue?